Sources/UIKitBackend/UIKitBackend+Control.swift

```diff
@@ -164,6 +170,36 @@ final class TappableWidget: ContainerWidget {
     }
 }
 
+@available(tvOS, unavailable)
+final class HoverableWidget: ContainerWidget {
+    private var hoverGestureRecognizer: UIHoverGestureRecognizer?
+
+    var hoverChangesHandler: ((Bool) -> Void)? {
+        didSet {
+            if hoverChangesHandler != nil && hoverGestureRecognizer == nil {
+                let gestureRecognizer = UIHoverGestureRecognizer(
+                    target: self,
+                    action: #selector(hoveringChanged(_:)))
+                child.view.addGestureRecognizer(gestureRecognizer)
+                self.hoverGestureRecognizer = gestureRecognizer
+            } else if hoverChangesHandler == nil, let hoverGestureRecognizer {
+                // should be impossible at the moment of implementation
+                // keeping it to be save in case of later changes
```

Comment:
It's safe to remove these comments, I don't think anyone will question that this is sensible (because the current guarantee that hoverChangesHandlers won't get set to `nil` is external to the HoverableWidget type)